docs/blog/2019-11-11-gazette-october/index.mdx

```diff
@@ -0,0 +1,123 @@
+tags:
+  - gazette
+---
+
+October means halloween! For your spooky reading pleasure the blog now has a dark mode. Dim the lights, pull up a ocmfy chair and we can begin.
+
+In September we posted our [Series A funding announcement](https://www.gatsbyjs.org/blog/2019-09-26-announcing-gatsby-15m-series-a-funding-round/), then headed to the UK for Gatsby Days London. Fear not, if you missed all the action you can [catch up by watching the videos](https://www.youtube.com/watch?v=UvO7h_KwgpE&list=PLCU2qJekvcN2f9rgS50xy7-4QBpgwid4G).
+
+![audience at Gatsby Days London](./gatsby-days.jpg)
+
+## Team highlights
+
+On with some features and updates from the team.
+
+First up is something we're calling _structured logging_. We've done some behind the scenes re-working of Gatsby's logging code. Gatsby now keeps an internal list of messages, and can output those messages in different formats. As an example, run `gatsby build --json` to see the your build logs output as JSON instead of the more usual log messages.
+
+In future this will enable things like a Gatsby desktop app. [Have a nose at the PR to see what went into this release](https://github.com/gatsbyjs/gatsby/pull/14973). As a result of this work there are have been a few fixes and enhancements to the standard CLI output.
+
+### Is Gatsby translated yet?
+
+Following on [from an RFC](https://github.com/gatsbyjs/rfcs/pull/42) in September, there are now 10 in-progress translations for Gatsby's documentation. Want to get involved? Check out [the new Gatsby Docs Translation Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/). Thanks to Nat Alison (twitter url here) for driving this project!
+
+translation scripts https://github.com/gatsbyjs/gatsby/pull/18893
+
+### Overview of the Gatsby Build Process
+
+Have you ever wondered what Gatsby _does_ when you run a build? Now there's a new docs page that runs through the details. [Overview of the Gatsby Build Process](https://www.gatsbyjs.org/docs/overview-of-the-gatsby-build-process/) is the place to go. I particularly like [the diagram that explains the different layers of Gatsby](https://www.gatsbyjs.org/docs/overview-of-the-gatsby-build-process/#what-happens-when-you-run-gatsby-build).
+
+### Schema Customization Examples
+
+We're gradually moving official plugins over to use the schema customisation APIs added earlier this year. See [the changes to `gatsby-transformer-sharp` for an example](https://github.com/gatsbyjs/gatsby/pull/18871). If you're interested in getting started with this, there's a little-known plugin that will write out the schema for your inferred data. See https://www.gatsbyjs.org/packages/gatsby-plugin-schema-snapshot/
+
+### Offline plugin
+
+The [offline plugin has been updated to handle more edge cases](https://github.com/gatsbyjs/gatsby/pull/17590). Now the plugin will revert back to the network if a visitor disables JavaScript after the site has already been cached. Thanks to @davidbailey00.
```

Comment:
Not sure if we usually do this or not, but consider linking @davidbailey00 to the GitHub profile?

Reply:
Yup, we usually _do_ that. This was oversight! Thanks, Flo ❤️